cpp/src/io/orc/reader_impl.hpp

```diff
@@ -176,7 +176,8 @@ class reader::impl {
    */
   column_buffer&& assemble_buffer(const int32_t orc_col_id,
                                   std::vector<std::vector<column_buffer>>& col_buffers,
-                                  const size_t level);
+                                  const size_t level,
+                                  rmm::cuda_stream_view stream);
```

Comment:
Should this have a default, or will we always be calling it internally with a stream?

Reply:
Yes, this is only for internal use only.